src/index.html

```diff
@@ -7,7 +7,107 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <section class="stars--0">
+      <div class="stars__star s1">
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">  
+      </div>
+
+      <div class="stars__star">
+      </div>
+
+      <div class="stars__star">
+      </div>
+    </section>
+    
+    <section class="stars--1">
+      <div class="stars__star image_container_active s1">   
```

Comment:
delete extra classes like this image_container_active and use :nth-child. If you have questions please ask them in chat